llvm/lib/LTO/LTO.cpp

```diff
@@ -1448,6 +1435,10 @@ Error ThinBackendProc::emitFiles(
 }
 
 namespace {
+// Base class for ThinLTO backends that perform code generation and insert the
+// generated files back into the link. This class exists primarily to share
+// common member variables, avoiding duplication between backend
+// implementations.
```

Comment:
Do we need to explicitly state the benefits of using class inheritance?

Reply:
Probably now. I was thinking of Teresa's queries on why this base class was needed. I will remove the second sentence from the comment.